website/content/partials/known-issues/internal-error-namespace-missing-policy.mdx

```diff
@@ -9,4 +9,53 @@ This impacts all auth methods.
 - 1.14.4 and 1.14.5
 - 1.15.0 and 1.15.1
 
-A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
\ No newline at end of file
+A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
+
+### Workaround
+During authentication, Vault will derive any additional policies that should be inherited based on any groups an entity is a member of. If Vault attempts to attach a policy to a token from a different namespace than the one handling the authentication, _and_ that policy does not exist, Vault will return an internal error. The way to resolve this issue is to either add the policy, or to delete the group policy mapping referring to the non-existent policy.
+
+Below is an example of a userpass auth method that is failing because of a
+missing group policy from another namespace.
+
+```
+# Failed login
+$ vault login -method=userpass username=user1 password=123
+Error authenticating: Error making API request.
+
+URL: PUT http://127.0.0.1:8200/v1/auth/userpass/login/user1
+Code: 500. Errors:
+
+* internal error
+
+# Identify the entity and group IDs
+$ vault read -format=json identity/entity/name/user1 | jq '{"entity_id": .data.id, "group_ids": .data.group_ids} '
+{
+  "entity_id": "420c82de-57c3-df2e-2ef6-0690073b1636",
+  "group_ids": [
+    "6cb152b7-955d-272b-4dcf-a2ed668ca1ea"
+  ]
+}
+
+# Fetch the group from the other namespace
+$ vault read -format=json -namespace=ns1 identity/group/id/6cb152b7-955d-272b-4dcf-a2ed668ca1ea | jq '.data.policies'
+[
+  "group_policy"
+]
+
+# The only policy in this namespace is default (group_policy is missing)
+$ vault policy list -namespace=ns1
+default
+
+# Option 1: Remove the missing policy from the group
+$ vault write -namespace=ns1 identity/group/id/6cb152b7-955d-272b-4dcf-a2ed668ca1ea name="test" policies="default"
+
+# OR Option 2: Create the missing policy
+$ vault policy write -namespace=ns1 group_policy - << EOF
+    path "secret/data/*" {
+	    capabilities = ["create", "update"]
+    }
+EOF
+
+# Login should succeed now
+$ vault login -method=userpass username=user1 password=123`
+```
```

Comment:
````suggestion

To fix the problem, we can either remove the missing policy from the
`6cb152b7-955d-272b-4dcf-a2ed668ca1ea` group or create the missing policy under
the `ns1` namespace.

<Tabs>

<Tab heading="Remove the group policy">

To remove `group_policy` from group ID `6cb152b7-955d-272b-4dcf-a2ed668ca1ea`,
use the `vault write` command to set the applicable policies to just include
`default`:

```shell-session
$ vault write                                             \
  -namespace=ns1                                          \
  identity/group/id/6cb152b7-955d-272b-4dcf-a2ed668ca1ea  \
  name="test"                                             \
  policies="default"
```

</Tab>

<Tab heading="Add the policy to ns1">

To create the missing policy, use `vault policy write` and define the
appropriate capabilities:

```shell-session
$ vault policy write -namespace=ns1 group_policy - << EOF
    path "secret/data/*" {
	    capabilities = ["create", "update"]
    }
EOF
```

</Tab>
</Tabs>

Verify the fix by re-running the login command:

<CodeBlockConfig hideClipboard>

```shell-session
$ vault login -method=userpass username=user1 password=123`
```

</CodeBlockConfig>
````

Style correction: make key elements easy to scan, avoid documenting in code comments, use one command per code block